build.sbt

```diff
@@ -222,13 +172,21 @@ lazy val `benchmarks-vnext` = project
 
 lazy val `marshallers-jodatime` = project
   .in(file("modules/marshallers/jodatime"))
-  .dependsOn(common % "compile->compile;test->test")
-  .dependsOn(channel % "compile->compile;test->test")
-  .dependsOn(`internal-core` % "compile->compile;test->test")
-  .dependsOn(testing % "test->test")
+  .dependsOn(`rpc-service`)
   .settings(moduleName := "mu-rpc-marshallers-jodatime")
   .settings(marshallersJodatimeSettings)
 
+///////////////
+//// TESTS ////
+///////////////
+
+lazy val tests = project
+  .in(file("modules/tests"))
+  .dependsOn(coreModulesDeps: _*)
+  .dependsOn(testing % "test->test")
```

Comment:
I don't think you need this dependency? All the tests from the `testing` module have been moved to the `tests` module.

Reply:
https://github.com/higherkindness/mu-scala/pull/900/commits/c4e9d6210c93dec62723cfb49e8c52c10daf05d3